docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -1171,10 +1171,46 @@ delegate :foo, to: :bar, allow_blank: true
 delegate :foo, to: :bar, allow_nil: true
 ----
 
+== Rails/DurationArithmetic
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| Yes
+| 2.13
+| -
+|===
+
+This cop checks if Duration as added to or subtracted from Time.current.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+Time.current - 1.minute
+Time.current + 2.days
+
+# fair
+Date.yesterday + 3.days
```

Comment:
I personally prefer `3.days.since(Date.yesterday)`, but sometimes it's more than just two, and:
 - it gets unweildy `1.minute.until(3.days.since(Date.yesterday))` vs `Date.yesterday + 3.days - 1.munute`
 - pattern's complexity would skyrocket